core/trino-main/src/main/java/io/trino/sql/planner/OptimizerConfig.java

```diff
@@ -756,4 +758,17 @@ public OptimizerConfig setUseExactPartitioning(boolean useExactPartitioning)
         this.useExactPartitioning = useExactPartitioning;
         return this;
     }
+
+    public boolean isFuseSubPlan()
+    {
+        return fuseSubPlan;
+    }
+
+    @Config("optimizer.fuse-sub-plan")
```

Comment:
Maybe `optimizer.combine-similar-sub-plans`
Fuse or fusion is not a common term